src/Shared/@components/Input/Input.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import s from './Input.module.scss';
+
+export const Input = props => {
+  return (
+    <div className={s.root}>
```

Comment:
Please, add `className`, so we can add some outside styles. For now use
```suggestion
    <div className={`${s.root} {className}`}>
```

Reply:
Шо